python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -594,3 +594,24 @@ def test_groupby_unique_lists():
     dd.assert_eq(
         gdf.groupby("a").b.unique(), gddf.groupby("a").b.unique().compute(),
     )
+
+
+@pytest.mark.parametrize("agg", ["first", "last"])
+def test_groupby_first_last(agg):
+    pdf = pd.DataFrame(
+        {"a": [2, 1, 2, 1, 1, 3], "b": [None, 1, 2, None, 2, None]}
```

Comment:
Add more test cases as in cudf /test_groupby

Reply:
Good catch! Added more tests here